scripts/jest/jest.config.js

```diff
@@ -1,26 +1,31 @@
+const ignores = ['node_modules', '__fixtures__', '__mocks__', 'dist', 'lib'];
```

Comment:
where are `'__fixtures__'` and `lib` coming from? do we have those in this repo?